src/components/CorporateCampaign/CampaignStatus.vue

```diff
@@ -4,33 +4,38 @@
 		:class="{
 			'campaign-status--loading': loadingPromotion,
 			'campaign-status--error': !promoApplied && promoErrorMessage && !loadingPromotion,
-			'campaign-status--success': promoApplied && !promoErrorMessage,
+			'campaign-status--success': (promoApplied && !promoErrorMessage) || statusMessageOverride,
 		}"
 	>
 		<div class="small-12 large-8 columns">
 			<div class="campaign-status__message">
-				<template v-if="loadingPromotion && !promoApplied && !promoErrorMessage">
-					<kv-loading-spinner />
-					<span>Validating Promotion</span>
+				<template v-if="statusMessageOverride">
+					<span v-html="statusMessageOverride"></span>
```

Comment:
could this just been a template string instead of v-html. Seeing as it will come out of a text setting and has no processing? Or do you think we could support basic html in that text string that would make this necessary?

Reply:
Good call, I don't think we need to have html in there.